vizro-ai/pyproject.toml

```diff
@@ -21,7 +21,7 @@ dependencies = [
   "langchain>=0.1.0, <0.3.0",  # TODO update to pydantic v2 and remove upper bound
   "langgraph>=0.1.2, <0.2.17",  # TODO update to pydantic v2 and remove upper bound (latest version break pydantic v1 compatibility)
   "python-dotenv>=1.0.0",  # TODO decide env var management to see if we need this
-  "vizro>=0.1.23",
+  "vizro<=0.1.29",  # Limited to current version of vizro, until the pydantic V2 changes of the main repo have been into here
```

Comment:
@nadijagraca @maxschulz-COL
Is the upper bound here set due to concerns that Pydantic V2 might negatively impact (or even break) the performance of Vizro-AI?